src/integrationtests/java/com/aws/greengrass/integrationtests/ipc/AuthorizeClientDeviceActionTest.java

```diff
@@ -125,13 +125,18 @@ void GIVEN_brokerWithAuthorizedAction_WHEN_AuthorizeClientDeviceAction_THEN_retu
                     .withOperation("some-action")
                     .withResource("some-resource")
                     .withClientDeviceAuthToken("some-token");
-            when(deviceAuthClient.canDevicePerform(any())).thenReturn(true);
+            AuthorizationRequest authzReq = AuthorizationRequest.builder()
```

Comment:
What you have here is good - keep this. You want to ensure that the IPC layer passes the correct arguments, and this achieves that